src/docs/upgrading.dox

```diff
@@ -1,5 +1,15 @@
 /*! @page upgrading Upgrading WiredTiger applications
 
+</dl><hr>
+@section version_1001 Upgrading to Version 10.0.1
+<dl>
+
+<dt>Background sync APIs</dt>
+<dd>
+The WT_SESSION::background_sync API and the sync=background configuration option to
+WT_SESSION::log_flush and WT_SESSION::commit_transaction have all been removed from WiredTiger.
+</dd>
```

Comment:
Should this be `WT_SESSION::transaction_sync`?

(FTR, I think the upgrading document is just going away, I should remove it and push that change so it's less confusing.)

Reply:
Good catch. Fixed.